### PR TITLE
Fixes Source Tags

### PR DESCRIPTION
**Pull request by jmagak**
**Version(s):** 1.2

**Issue:** https://issues.redhat.com/browse/RHIDP-3484

**Link to docs preview:**

**Reviews:**
- [ ] SME: @ mention assignee
- [ ] QE: @ mention assignee
- [ ] Docs review: @ mention assignee

**Additional information:**

### CODE REVIEW

**Comment by gwynnemonahan** [comment]
Hey @jmagak ,

To help with reviews, will you add the Version this PR applies to, and a link to the Jira issue associated with this PR?

Thanks!

**Comment by themr0c** [comment]
Your pull request should go to https://github.com/redhat-developer/red-hat-developers-documentation-rhdh rather than to your fork.

File: modules/installation/proc-deploy-rhdh-instance-eks.adoc [hunk 1]
```diff
@@ -62,7 +62,7 @@ Ensure that you use a unique value of `BACKEND_SECRET` for each {product-short}
 
 You can use the following command to generate a key:
 
-[source]
+[source,yaml]
```
Comment (themr0c) [inline modules/installation/proc-deploy-rhdh-instance-eks.adoc]:
```suggestion
[source,terminal]
```

File: modules/installation/proc-deploy-rhdh-instance-eks.adoc [hunk 2]
```diff
@@ -71,7 +71,7 @@ node-p'require("crypto").randomBytes(24).toString("base64")'
 . To enable pulling the PostgreSQL image from the {company-name} Ecosystem Catalog, add the image pull secret in the default service account within the namespace where the {product-short} instance is being deployed:
 +
 --
-[source]
+[source,yaml]
```
Comment (themr0c) [inline modules/installation/proc-deploy-rhdh-instance-eks.adoc]:
```suggestion
[source,terminal]
```

File: modules/installation/proc-rhdh-deploy-eks-helm.adoc [hunk 3]
```diff
@@ -47,7 +47,7 @@ The created pull secret is used to pull the {product-short} images from the {com
 
 . Create a file named `values.yaml` using the following template:
 +
-[source,yaml,subs="attributes+"]
+[source,terminal,subs="attributes+"]
```
Comment (themr0c) [inline modules/installation/proc-rhdh-deploy-eks-helm.adoc]:
```suggestion
[source,yaml,subs="attributes+"]
```

File: modules/installation/proc-rhdh-deploy-eks-operator.adoc [hunk 4]
```diff
@@ -53,7 +53,7 @@ The created pull secret is used to pull the {product-short} images from the {com
 . Create a `CatalogSource` resource that contains the Operators from the {company-name} Ecosystem:
 +
 --
-[source,subs="attributes+"]
+[source,yaml,subs="attributes+"]
```
Comment (themr0c) [inline modules/installation/proc-rhdh-deploy-eks-operator.adoc]:
```suggestion
[source,terminal,subs="attributes+"]
```

File: modules/installation/proc-rhdh-deploy-eks-operator.adoc [hunk 5]
```diff
@@ -73,7 +73,7 @@ EOF
 . Create an `OperatorGroup` resource as follows:
 +
 --
-[source]
+[source,yaml]
```
Comment (themr0c) [inline modules/installation/proc-rhdh-deploy-eks-operator.adoc]:
```suggestion
[source,terminal]
```

File: modules/installation/proc-rhdh-deploy-eks-operator.adoc [hunk 6]
```diff
@@ -87,7 +87,7 @@ EOF
 . Create a `Subscription` resource using the following code:
 +
 --
-[source,subs="attributes+"]
+[source,yaml,subs="attributes+"]
```
Comment (themr0c) [inline modules/installation/proc-rhdh-deploy-eks-operator.adoc]:
```suggestion
[source,terminal,subs="attributes+"]
```

File: modules/installation/proc-rhdh-deploy-eks-operator.adoc [hunk 7]
```diff
@@ -109,7 +109,7 @@ EOF
 . Run the following command to verify that the created Operator is running:
 +
 --
-[source]
+[source,yaml]
```
Comment (themr0c) [inline modules/installation/proc-rhdh-deploy-eks-operator.adoc]:
```suggestion
[source,terminal]
```

File: modules/installation/proc-rhdh-deploy-eks-operator.adoc [hunk 8]
```diff
@@ -120,7 +120,7 @@ If the operator pod shows `ImagePullBackOff` status, then you might need permiss
 ====
 You can include the required secret name in the `deployment.spec.template.spec.imagePullSecrets` list and verify the deployment name using `kubectl get deployment -n rhdh-operator` command:
 
-[source]
+[source,yaml]
```
Comment (themr0c) [inline modules/installation/proc-rhdh-deploy-eks-operator.adoc]:
```suggestion
[source,terminal]
```

File: modules/installation/proc-rhdh-deploy-eks-operator.adoc [hunk 9]
```diff
@@ -133,7 +133,7 @@ kubectl -n rhdh-operator patch deployment \
 .. Edit the `backstage-default-config` ConfigMap in the `rhdh-operator` namespace using the following command:
 +
 --
-[source]
+[source,yaml]
```
Comment (themr0c) [inline modules/installation/proc-rhdh-deploy-eks-operator.adoc]:
```suggestion
[source,terminal]
```

File: modules/installation/proc-rhdh-deploy-eks-operator.adoc [hunk 10]
```diff
@@ -210,7 +210,7 @@ Wait for a few minutes until the changes are automatically applied to the operat
 . Clone the Operator repository to your local machine using the following command:
 +
 --
-[source]
+[source,yaml]
```
Comment (themr0c) [inline modules/installation/proc-rhdh-deploy-eks-operator.adoc]:
```suggestion
[source,terminal]
```

File: modules/installation/proc-rhdh-deploy-eks-operator.adoc [hunk 11]
```diff
@@ -219,7 +219,7 @@ git clone --depth=1 https://github.com/redhat-developer/rhdh-operator.git rhdh-o
 . Run the following command and generate the deployment manifest:
 +
 --
-[source]
+[source,yaml]
```
Comment (themr0c) [inline modules/installation/proc-rhdh-deploy-eks-operator.adoc]:
```suggestion
[source,terminal]
```

File: modules/installation/proc-rhdh-deploy-eks-operator.adoc [hunk 12]
```diff
@@ -230,7 +230,7 @@ The previous command generates a file named `rhdh-operator-<VERSION>.yaml`, whic
 . Run the following command to apply replacements in the generated deployment manifest:
 +
 --
-[source]
+[source,yaml]
```
Comment (themr0c) [inline modules/installation/proc-rhdh-deploy-eks-operator.adoc]:
```suggestion
[source,terminal]
```

File: modules/installation/proc-rhdh-deploy-eks-operator.adoc [hunk 13]
```diff
@@ -296,7 +296,7 @@ sed -i "s/backstage-controller-manager/rhdh-controller-manager/g" rhdh-operator-
 .. Replace the default images with the images that are pulled from the {company-name} Ecosystem:
 +
 --
-[source,subs="attributes+"]
+[source,yaml,subs="attributes+"]
```
Comment (themr0c) [inline modules/installation/proc-rhdh-deploy-eks-operator.adoc]:
```suggestion
[source,terminal,subs="attributes+"]
```

File: modules/installation/proc-rhdh-deploy-eks-operator.adoc [hunk 14]
```diff
@@ -349,7 +349,7 @@ spec:
 . Apply the manifest to deploy the operator using the following command:
 +
 --
-[source]
+[source,yaml]
```
Comment (themr0c) [inline modules/installation/proc-rhdh-deploy-eks-operator.adoc]:
```suggestion
[source,terminal]
```

File: modules/installation/proc-rhdh-deploy-eks-operator.adoc [hunk 15]
```diff
@@ -358,7 +358,7 @@ kubectl apply -f rhdh-operator-VERSION.yaml
 . Run the following command to verify that the Operator is running:
 +
 --
-[source]
+[source,yaml]
```
Comment (themr0c) [inline modules/installation/proc-rhdh-deploy-eks-operator.adoc]:
```suggestion
[source,terminal]
```

**Comment by jmagak** [comment]
Closing this PR due to a conflict in the repository.

**Pull request closed without merging**